doc/github-workflow.md

```diff
@@ -0,0 +1,35 @@
+
+# GitHub workflow guidelines
+
+## Project Boards
+- One JValue-Scope Project board
+- One for each repository, linked to issues
+- One project board for each student project in the corresponding repo, possibly linked to issues 
+  
+## Issues
+- One issue board for each repository
+- Issues should be tagged appropriately to enable external contributions
+
+## External Contributions
+- External contributions are to be encouraged
+- External contributors may stage issues or work on existing ones
```

Comment:
Self-staged issues may be discussed with the admin teams in order to ensure later integration.

Reply:
Irgendwie sowas noch?